<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Steghide-GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>858</width>
      <height>574</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="FilesAndFolders_MainPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLabel" name="Label_SpaceforFiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Free space for secret files:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="Label_Secret_Files">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>(Super Secret Files)</string>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>Hidden files:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="3" colspan="2">
      <widget class="QTreeView" name="FilesAndFoldersTreeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="Label_Carrier_Files">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Carrier files:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QProgressBar" name="FreeSpaceProgressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="2" colspan="2">
      <widget class="QTableView" name="FileTableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>180</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <layout class="QGridLayout" name="MenuGridLayout">
       <item row="0" column="1">
        <widget class="QToolButton" name="AddFilesButton">
         <property name="toolTip">
          <string>Add a file to the list of files to be hidden</string>
         </property>
         <property name="text">
          <string>Add Files</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/add.png</normaloff>:/resources/images/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QToolButton" name="HelpButton">
         <property name="toolTip">
          <string>Pretty self explainatory</string>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/help.png</normaloff>:/resources/images/help.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="ExtractButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Extract files from carrier</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Extract Files</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/extract.png</normaloff>:/resources/images/extract.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="ReloadButton">
         <property name="toolTip">
          <string>Refresh the file explorer</string>
         </property>
         <property name="text">
          <string>Reload</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/reload.png</normaloff>:/resources/images/reload.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="EncodeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Place files into carrier</string>
         </property>
         <property name="text">
          <string>Hide Files</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/encode.png</normaloff>:/resources/images/encode.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="SettingsButton">
         <property name="toolTip">
          <string>Configure StegHide-GUI</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/settings.png</normaloff>:/resources/images/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="CheckFileButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Check file for StegHide information</string>
         </property>
         <property name="text">
          <string>Check Contents</string>
         </property>
         <property name="icon">
          <iconset resource="imageresources.qrc">
           <normaloff>:/resources/images/convert.png</normaloff>:/resources/images/convert.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
    <zorder>FreeSpaceProgressBar</zorder>
    <zorder>Label_SpaceforFiles</zorder>
    <zorder>line</zorder>
    <zorder>FileTableView</zorder>
    <zorder>Label_Secret_Files</zorder>
    <zorder>FilesAndFoldersTreeView</zorder>
    <zorder>FilesAndFolders_MainPath</zorder>
    <zorder>Label_Carrier_Files</zorder>
    <zorder></zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageresources.qrc"/>
 </resources>
 <connections/>
</ui>
